streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/resource/Container.java

```diff
@@ -0,0 +1,86 @@
+package org.ray.streaming.runtime.core.resource;
+
+import com.google.common.base.MoreObjects;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.ray.api.id.UniqueId;
+
+/**
+ * Resource manager unit abstraction.
+ * Container identifies the available resource(cpu,mem) and allocated slots.
+ */
+public class Container implements Serializable {
```

Comment:
Who create `container`? Does resource manager ask from ray or other external system such as yarn/k8s?  If external system, what's the api to interact with them?

Reply:
Now it is necessary to get the node of the ray cluster directly to create the container, and then pr will access the ```deploy``` api.